packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/02-installation/02-expo.mdx

```diff
@@ -24,60 +24,36 @@ So what did we install precisely?
 
 - `@stream-io/video-react-native-sdk` (SVRN) is Stream's Video SDK which contains UI components, hooks and util functions that will enable audio/video calls.
 - `@stream-io/react-native-webrtc` is a WebRTC module for React Native, SVRN depends on this dependency, it's components and utilities to render audio/video tracks and interact with the phone's media devices.
-- `react-native-device-info` is responsible for retrieving device info and sending it to the Stream Video servers.
 - `react-native-incall-manager` handles media-routes/sensors/events during an audio/video call.
 - `react-native-svg` provides SVG support to React Native, SVRN's components and it's icons are reliant on this dependency.
 - `@react-native-community/netinfo` - is used to detect the device's connectivity state, type and quality.
 - `@notifee/react-native` - is used to keep calls alive in the background on Android.
+
 ## Add config plugin
 
-Add the config plugin for `react-native-webrtc` to your `app.json` file:
+Add the config plugin for `@stream-io/video-react-native-sdk` and `react-native-webrtc` to your `app.json` file:
+
+The `@stream-io/video-react-native-sdk` handles the [Add Stream Video SDK's setup method](../react-native/#add-stream-video-sdks-setup-method) automatically and adds the appropriate android permissions such as `POST_NOTIFICATIONS`, `FOREGROUND_SERVICE`, `FOREGROUND_SERVICE_MICROPHONE`, `BLUETOOTH`, `BLUETOOTH_ADMIN` and `BLUETOOTH_CONNECT` to their respective files.
 
-```js title=app.json
+```js title=app.json {5-13}
```

Comment:
```suggestion
```js title=app.json
```

lets do ```// highlight-start and stop``` instead as recommended in the docusaurus docs. Line numbers are unmaintainable when we add more

Reply:
I can do it but I thought this is more reliable way of adding highlights in the code without adding additional lines.

Reply:
nah docusaurus will remove the comment.. so its fine